src/app.ts

```diff
@@ -2,22 +2,31 @@ import * as bodyParser from "body-parser";
 import express from "express";
 import { Routes } from "./routes";
 
+/**
+ * @class
```

Comment:
Any specific reason the comments are removed? It doesn't really make a difference, you can carry on however it suits you for now

Reply:
added them back